core/modules/measure.cc

```diff
@@ -132,21 +129,20 @@ void Measure::ProcessBatch(Context *ctx, bess::PacketBatch *batch) {
 
       bytes_cnt_ += batch->pkts()[i]->total_len();
 
-      rtt_hist_.Insert(diff);
+      rtt_hist_.AtomicInsert(diff);
       if (rand_.GetRealNonzero() <= jitter_sample_prob_) {
         if (unlikely(!last_rtt_ns_)) {
           last_rtt_ns_ = diff;
           continue;
         }
-        uint64_t jitter = absdiff(diff, last_rtt_ns_);
-        jitter_hist_.Insert(jitter);
+        int64_t jitter = diff - last_rtt_ns_;
+        jitter = jitter * ((jitter>0) - (jitter<0));
```

Comment:
Why not just use the `abs`
```suggestion
        int64_t jitter = abs(diff - last_rtt_ns_);
```
Of course, the above would need to include `<cstdlib>` if needed